docs/source/deployment/deploy_sre.md

```diff
@@ -18,6 +18,11 @@ Each project will have its own dedicated Secure Research Environment (SRE).
 azure:
   subscription_id: # ID of the Azure subscription that the TRE will be deployed to
   tenant_id: # Home tenant for the Azure account used to deploy infrastructure: `az account show`
+description: # A free-text description of your SRE deployment
+dockerhub:
+  access_token: # The password or personal access token for the above account. We strongly recommend using a Personal Access Token with permissions set to Public Repo Read-only
```

Comment:
```suggestion
  access_token: # The password or personal access token for the above account. We strongly recommend using a Personal Access Token with permissions set to Public Repo Read-only
```

"above account" reads oddly given this field is before `username`, which says "your Docker Hub account" - maybe put "your Docker Hub account" here and "the above account" after `username`